src/AWS.Deploy.Common/Recommendation.cs

```diff
@@ -200,10 +200,30 @@ public bool IsOptionSettingDisplayable(OptionSettingItem optionSetting)
                 var dependsOnOptionSettingValue = GetOptionSettingValue(dependsOnOptionSetting);
                 if (
                     dependsOnOptionSetting != null &&
-                    dependsOnOptionSettingValue != null &&
-                    !dependsOnOptionSettingValue.Equals(dependency.Value))
+                    dependsOnOptionSettingValue != null)
                 {
-                    return false;
+                    if (dependency.Operation != null)
+                    {
+                        if (dependency.Operation == PropertyDependencyOperationType.NotEmpty)
+                        {
+                            if (dependsOnOptionSetting.Type == OptionSettingValueType.List &&

```

Comment:
We should add maps (i.e. Environment Variables) as well.

Reply:
Done